django-website/wheelie/package.json

```diff
@@ -19,6 +19,7 @@
   "devDependencies": {
     "gulp": "3.9.0",
     "wheelie": "0.1.3",
-    "wheelie-recipe": "0.2.1"
+    "wheelie-recipe": "0.2.1",
+    "normalize.css": "5.0.0"
```

Comment:
Normalize.css should be a production dependency. Remove normalize.css from devDependencies, then install it by running `npm install --save normalize.css` (or `bower install --save normalize.css` if we're still using Bower);